lib/main.js

```diff
@@ -10,6 +10,7 @@ let events = require("sdk/system/events");
 let runtime = require("sdk/system/runtime");
 let utils = require("sdk/window/utils");
 let system = require("sdk/system");
+let _ = require("sdk/l10n").get;
```

Comment:
Is this abbreviation common in l10n implementations? Writing out the full 'l10n.get(...)' would be more clear to me.